src/components/activity/ActivityDirectivesTable.svelte

```diff
@@ -219,3 +293,7 @@
     <ContextMenu.Separator />
   </svelte:fragment>
 </BulkActionDataGrid>
+
+{#if showBulkShiftDialog}
```

Comment:
Our pattern for modals is a bit boilerplate-y but what we do is run everything through the `effects.ts` file and add modals to the `modals.ts` file so that these functions can be called from anywhere and the modals don't need to be mounted in any specific component.

Reply:
Would also let us more easily call `shift` on a directive from the activity timeline.

Reply:
Updated to use `effects` + standard modals pattern, reviewed with @duranb